src/Stratis.Bitcoin.Features.Wallet/TransactionBuildOptions.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Wallet
+{
+    /// <summary>
+    /// Options object to be passed to <see cref="WalletTransactionHandler"/>.
+    /// </summary>
+    public class TransactionBuildOptions
+    {
+        /// <summary>
+        /// Wallet account to use to build transaction. This is the account from which UTXOs will be selected. Required.
+        /// </summary>
+        public WalletAccountReference WalletAccountReference { get; set; }
```

Comment:
It could be - I'd rather either have the whole thing immutable or none of it. 

There could feasibly be situations where you want to build up an options object then try different wallet references afterwards. 

Maybe there is a specific BuildOptions setting that you keep because it makes a lot of sense for your individual application, and you wish to change only the wallet used.